api/src/main/java/jakarta/data/metamodel/SortableAttribute.java

```diff
@@ -42,11 +44,30 @@ public interface SortableAttribute<T> extends Attribute<T> {
      */
     Sort<T> asc();
 
+    default Restriction<T> between(Comparable<Object> min, Comparable<Object> max) {
+        return Restrict.between(min, max, name());
+    }
+
     /**
      * Obtain a request for a descending {@link Sort} based on the entity attribute.
      *
      * @return a request for a descending sort on the entity attribute.
      */
     Sort<T> desc();
 
+    default Restriction<T> greaterThan(Comparable<Object> value) {
```

Comment:
Should this comparable be `Comparable<? extends Object> value`, I am not sure here.

Reply:
I think it should probably just be `T`.